content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx

```diff
@@ -310,9 +310,7 @@ This relation expresses the following:
 - "a user can be followed by zero or more users"
 - "a user can follow zero or more users"
 
-Note that this many-to-many-relation is [implicit](many-to-many-relations#implicit-many-to-many-relations). This means Prisma maintains a [relation table](/concepts/components/prisma-schema/relations/many-to-many-relations#relation-tables) for it in the underlying database.
-
-If you need the relation to hold other fields, on relational databases you can create an [explicit](many-to-many-relations#explicit-many-to-many-relations) many-to-many self relation as well. The explicit version of the self relation shown previously is as follows:
+Note that for relational databases, this many-to-many-relation is [implicit](many-to-many-relations#implicit-many-to-many-relations). This means Prisma maintains a [relation table](/concepts/components/prisma-schema/relations/many-to-many-relations#relation-tables) for it in the underlying database. If you need the relation to hold other fields, you can create an [explicit](many-to-many-relations#explicit-many-to-many-relations) many-to-many self relation as well. The explicit version of the self relation shown previously is as follows:
```

Comment:
Why remove the empty line here? Before it was clearer what is implicit (first paragraph) and explicit (second paragraph).

Reply:
Good question. I've put it back